modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -20,10 +21,14 @@
 
 import java.util.Objects;
 
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
 /**
  * @author Marco Leo
  * @author Brian Wing Shun Chan
  */
+@Component(service = ObjectDefinition.class)
```

Comment:
We cannot do this... The ObjectDefinition can not be registered as an OSGi component... What is the purpose of that? Should we just have one ObjectDefinition in the whole portal? It is the implementation of the ObjectDefinition database DTO

Reply:
SF doesn't allow me to put @Reference without @Component, I added it cause it was easier for me than changing persistence to ObjectDefinitionUtil method in tests (it is implemented as static)

Reply:
I reread your comment, and yeah, it is quite not what we're trying to achieve xD Honestly, I did not even think what @\Component tag implies, SF told me to add it, so I did 😅